heliopy/data/custom/units_dict.py

```diff
@@ -0,0 +1,33 @@
+"""
+CDF Unit Dictionary
+-------------------
+Method to obtain the unit denoted by the strings
+inside the CDF files in the UNIT attribute.
+"""
+
+import astropy.units as u
+from collections import OrderedDict
+
+
+def get(unit_string):
+    units = OrderedDict([('ratio', u.dimensionless_unscaled),
+                        ('#/cc', u.cm**-3),
+                        ('#/cm^3', u.cm**-3),
+                        ('cm^{-3}', u.cm**-3),
+                        ('ionic charge', u.electron),
+                        ('earth radii', u.earthRad),
+                        ('Re', u.earthRad),
+                        ('Degrees', u.deg),
+                        ('degrees', u.deg),
+                        ('#/{cc*(cm/s)^3}', (u.cm**3 * (u.cm / u.s)**3)**-1),
+                        ('Angle', u.deg),
```

Comment:
Angles might not always be degrees (they could be radians), so best not to have this mapping in general and to do it on a case by case basis for each function instead.

Reply:
Got it!